Makefile

```diff
@@ -49,7 +49,7 @@ help::
 	@echo ' make run-cc-tests'
 	@echo ' '
 	@echo 'To build client-server performance test programs and run performance test'
-	@echo ' make clean && CXX=g++ LD=g++ make client-server-perf-test'
+	@echo ' make clean && CC=gcc LD=g++ make client-server-perf-test'
```

Comment:
Minor correction. We just need `gcc` for these programs for now; no need for `g++`.